apps/remix-ide/src/app/components/plugin-manager-component.js

```diff
@@ -102,6 +102,57 @@ class PluginManagerComponent extends ViewPlugin {
     return this.appManager.actives.includes(name)
   }
 
+  async registerFlattener (event) {
```

Comment:
This should preferably be in `remixAppManager.js` 
@ioedeveloper @Aniket-Engg @bunsenstraat any opinion regarding where this should be added?

Reply:
Yes, i agree.